### ISSUE TITLE
Publish on PyPi

### ISSUE THREAD

**Issue by roll** [frictionlessdata/datapackage-py#71]
The last piece of puzzle.

Already on PyPi:
```
tabulator
jsontableschema
jsontableschema-bigquery
jsontableschema-sql
```

**Comment by roll** [comment]
@vitorbaptista 
WDYT?)

**Comment by pwalsh** [comment]
@roll it is related to https://github.com/trickvi/datapackage/issues/70

Hey @trickvi can I help with moving and/or closing issues over there?

**Comment by vitorbaptista** [comment]
Done! Released version 0.6.0 on https://pypi.python.org/pypi/datapackage :beers:

Thanks all!
